src/actions/Preview/PreviewWindow/PreviewWindow.jsx

```diff
@@ -0,0 +1,46 @@
+import { useState, useRef, useEffect } from 'react';
+import cls from './styles.module.css';
+
+import { getStoreValue } from '../../../store';
+import { PREVIEW_PORT } from '../../../constants/buckets';
+
+import getLocalIlloSlugs from '../../../utils/get-local-illo-slugs';
+
+const baseURL = `http://localhost:${PREVIEW_PORT}/`;
+
+function PreviewWindow() {
+  const [url, setURL] = useState(baseURL);
+  const [illos, setIllos] = useState(null);
+  const iframeRef = useRef(null);
+
+  useEffect(() => {
+    async function getSlugs() {
+      const projectSlug = await getStoreValue('active-project');
+      const illoSlugs = await getLocalIlloSlugs(projectSlug);
+      const illoURL = `http://localhost:${PREVIEW_PORT}/${illoSlugs[0]}/ai2html-output/index.html`;
```

Comment:
This line is too long, should try something like

```suggestion
      const illoURL = `http://localhost:${PREVIEW_PORT}/${illoSlugs[0]}/`
        + `ai2html-output/index.html`;
```